parlai/scripts/convo_render.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import json
+import random
+import argparse
+import tempfile
+
+# Constants
+END_OF_CONVO = "EOC"
+CHROME_PATH = (
+    '/Applications/Google\ Chrome.app/Contents/MacOS//Google\ Chrome'
+)  # noqa: W605
```

Comment:
Make it `r'/Applications/Google\ Chrome.app/Contents/MacOS//Google\ Chrome'` (note the prefix `r` to indicate a raw string). This will get rid of the flake issue.